beets/dbcore/types.py

```diff
@@ -190,6 +190,38 @@ class String(Type):
     query = query.SubstringQuery
 
 
+class StringList(Type):
+    """A List of Unicode strings type, stored in a string
+    """
+    sql = u'TEXT'
+    query = query.SubstringQuery
+
+    def from_sql(self, sql_value):
+        return self.parse(sql_value)
+
+    def to_sql(self, value):
+        return self.format(value)
```

Comment:
Interesting! We've never used `parse` and `format` to interact with the underlying SQL value, but I suppose this makes sense. But maybe null characters for the SQL storage would be a bit safer?